runtime/common/src/migrations.rs

```diff
@@ -565,6 +565,88 @@ impl<T: pallet_scheduler::Config> Migration for SchedulerMigrationV3<T> {
 	}
 }
 
+/// BaseFee pallet, set Elasticity to zero.
+/// This migration needs to be applied before or at the same time we introduce:
+/// https://github.com/paritytech/frontier/pull/794
+pub struct BaseFeeElasticity<T>(PhantomData<T>);
+impl<T: BaseFeeConfig> OnRuntimeUpgrade for BaseFeeElasticity<T> {
+	/// Run a standard pre-runtime test. This works the same way as in a normal runtime upgrade.
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		let module: &[u8] = b"BaseFee";
+		// Elasticity storage value
+		{
+			let item: &[u8] = b"Elasticity";
+			let value = get_storage_value::<Permill>(module, item, &[]).unwrap_or(Permill::zero());
+			Self::set_temp_storage(value, "elasticity_pre_upgrade");
+		}
+
+		Ok(())
+	}
+
+	fn on_runtime_upgrade() -> Weight {
+		let module: &[u8] = b"BaseFee";
+		let db_weights = T::DbWeight::get();
+		let mut weight: Weight = 1 * db_weights.read;
+		// Elasticity storage value
+		{
+			let item: &[u8] = b"Elasticity";
+			let current_value =
+				get_storage_value::<Permill>(module, item, &[]).unwrap_or(Permill::zero());
+			if !current_value.is_zero() {
+				// Set Elasticity to zero, which results in constant BaseFeePerGas
+				let write = pallet_base_fee::Pallet::<T>::set_elasticity_inner(Permill::zero());
```

Comment:
@tgmichel After this, maybe it's time to change the fixed gas price to the base-fee pallet?

Reply:
Yes thank you, will do that on a separate PR